### ISSUE TITLE
All headings of major sections should be navigatable by keyboard.

### ISSUE THREAD

**Issue by itaditya** [hasura/learn-graphql#46]
Ideal flow for Keyboard user should be-

Checkout Tutorials -> Frontend Tutorials -> React -> Vue ... -> Mobile Tutorials ... -> Testimonials -> What you will learn.

I can work on this if you're interested

**Comment by sigfriedCub1990** [comment]
Hello there, is @itaditya working on this issue?

**Comment by itaditya** [comment]
@praveenweb please review this PR https://github.com/hasura/learn-graphql/pull/69
